packages/babel-preset-env/src/index.js

```diff
@@ -301,10 +301,25 @@ export default declare((api, opts) => {
 
   const pluginUseBuiltIns = useBuiltIns !== false;
   const plugins = Array.from(pluginNames)
-    .map(pluginName => [
-      getPlugin(pluginName),
-      { spec, loose, useBuiltIns: pluginUseBuiltIns },
-    ])
+    .map(pluginName => {
+      if (
+        pluginName === "proposal-class-properties" ||
+        pluginName === "proposal-private-methods"
```

Comment:
I think we can use `classFeaturePlugins = ["proposal-class-properties", "proposal-private-methods", "proposal-private-property-in-object"]` to align with the class feature plugin factory. Although it is not included in preset-env yet but we are likely to forget adding them when `privateIn` reaches stage 3.